### PR TITLE
entrypoint script to give external ip as 0.0.0.0

### PR DESCRIPTION
**Pull request by Moteesh**
Changed the entrypoint script to give external ip as 0.0.0.0
    1. Why is this change necessary ?
           In order for istgt to listen on all the available pod ips for replicas to connect.
    2. How does this change address the issue ?
         With this change, istgt will listen on all available pod ips for connection from replicas
    3. How to verify this change ?
        Deploy istgt and connect replicas through pod ip or service endpoint ip.

Signed-off-by: moteesh <moteesh@gmail.com>

### CODE REVIEW

File: entrypoint-istgtimage.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ truncate -s $size $path/$volname
 cp /usr/local/etc/bkpistgt/istgt.conf /usr/local/etc/istgt/
 touch /usr/local/etc/istgt/auth.conf
 touch /usr/local/etc/istgt/logfile
-externalIP=127.0.0.1
+externalIP=0.0.0.0
```
[the remark this answers is not in the document]
Comment (jkryl) [reply entrypoint-istgtimage.sh]: I would be interested to know too. and it should be:
```
externalIP=...
export externalIP
```
or
```
export externalIP=...
```

[the remark this answers is not in the document]
Comment (Moteesh) [reply entrypoint-istgtimage.sh]: I believe it works both ways `export externalIP=...`
As per my understanding, istgt will use the externalIP to listen for replica connections. In this case it listens on all the ips of the container by making it as "0.0.0.0"

[the remark this answers is not in the document]
Comment (Moteesh) [reply entrypoint-istgtimage.sh]: I copied this from istgt src code repo. Not sure which one to use, so used it the way they have been using in their test scripts

[the remark this answers is not in the document]
Comment (jkryl) [reply entrypoint-istgtimage.sh]: it certainly does not make any harm but it looks weird. you assign the same variable twice, which does not make sense. You should either assign it once and export it (without assigning to it again) or export it as part of assignment.

**Pull request merged**